Client/Frontend/Browser/EnhancedTrackingProtection/EnhancedTrackingProtectionDetailsVC.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  EnhancedTrackingProtectionDetalsVC.swift
+//  Client
+//
+//  Created by Roux Buciu on 2021-08-04.
+//  Copyright © 2021 Mozilla. All rights reserved.
+//
```

Comment:
There's some of these you'll need to replace with the license throughout the PR.

Reply:
oops. totally forgot to change those. LOL Will do.